config/twill/block_editor.php

```diff
@@ -68,6 +68,8 @@
         'upload_manifest_file',
     ],
     'browser_route_prefixes' => [
+        'digitalPublications' => 'collection.articlesPublications',
+        'publications' => 'collection.articlesPublications',
```

Comment:
Where is there a reference to a browser just named `publications`? Is this the browser for printed publications?